kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/RuntimeDataServiceBase.java

```diff
@@ -653,8 +653,13 @@ public ProcessInstanceCustomVarsList queryProcessesByVariables(String payload, S
             return convertToProcessInstanceCustomVarsList(advanceRuntimeDataService.queryProcessByVariables(convertToServiceApiQueryParam(filter.getAttributesQueryParams()),
                                                                                                             convertToServiceApiQueryParam(filter.getProcessVariablesQueryParams()),
                                                                                                             queryContext));
+        } else if (filter.getOwnersQueryParam() != null) {
+            return convertToProcessInstanceCustomVarsList(advanceRuntimeDataService.queryProcessByVariablesAndTask(convertToServiceApiQueryParam(filter.getAttributesQueryParams()),
+                                                                                                                   convertToServiceApiQueryParam(filter.getProcessVariablesQueryParams()),
+                                                                                                                   convertToServiceApiQueryParam(filter.getTaskVariablesQueryParams()),
+                                                                                                                   filter.getOwnersQueryParam(),
```

Comment:
You need to do a similar conversion from exposed `QueryParam` to engine `QueryParam` object as it's done in the above parameters above.